cli/mizu/configStruct.go

```diff
@@ -6,6 +6,13 @@ import (
 	"github.com/up9inc/mizu/cli/mizu/configStructs"
 )
 
+const (
+	AgentImageConfigName             = "agent-image"
+	MizuResourcesNamespaceConfigName = "mizu-resources-namespace"
```

Comment:
not related to this commit, but can you search for all "mizu-resources-namespace" accurances in the code 
they are mainly in errors or logs, and use this const there